website/content/labs/lab1.md

```diff
@@ -30,30 +43,34 @@ Log into https://mit.cs.uchicago.edu/. Under "Your projects", you should see an
   cd cs220
   ```
 
-* Inside that folder, create a file called `test.txt` with your full name.
+* Inside that folder, create a file called `README.md` with your full name.
 
-* On the GitLab project page (the one that says "The repository for this project is empty") there is pull-down list with "SSH" selected. Make sure to change this to "HTTPS".
+* On your repository's GitHub page, right under "Quick setup — if you’ve done this kind of thing before" there is a URL field with two buttons: HTTPS and SSH. Make sure to change this to "HTTPS".
 
-Now, from inside your CMSC 22000 directory, run the instructions for "Git global setup" and for "Existing folder". If any commands asks you to enter a username and password, just enter your CNetID and password. Don't worry about what each individual command does; we will be seeing what most of these commands do in this lab. However, if you encounter any error messages, please make sure to let us know.
+Now, from inside your CMSC 22000 directory, run the commands that appear under "...or create a new repository on the command line" *except* the first one (the one that starts with `echo`. 
 
-You can verify that your repository was correctly set up by going back to https://mit.cs.uchicago.edu/. If you go to your repository page, you should now see it contains a `test.txt` file. If you click on it, you can see its contents.
+*Note: If any commands asks you to enter a username and password, just enter your GitHub username and password* (do not enter your CNetID and password)
```

Comment:
might be worth noting that if they have 2FA enabled on their GH account, they'll need to set up a CLI token to be able to use HTTPS authentication. [here's](https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line) GH's instructions - for step 7 they'll just need the `repo` permission as suggested. might also be worth throwing in a mention to set `credential.helper` to `store` so they don't have to keep the token anywhere.

Reply:
(also - I personally use SSH auth with GH because I got sick of this flow. not sure if that's the right move for the class though 🤷‍♂)

Reply:
ssh is the bomb, screw https

Reply:
I really don't want to unleash SSH keys on them this early. Task 8 actually asks them to figure out how to avoid having to enter their password every time (which is in line with having this lab also being about them figuring stuff out on their own)